app/twitch/create-game-element.js

```diff
@@ -0,0 +1,23 @@
+export default function createGameElement(game) {
+  const item = document.createElement('li');
+  item.classList.add('game-item');
+
+  item.innerHTML = `
+  <img src="" alt="" class="game-item__pic"></img>
+ <div className="frame">
+   <h3 class="game-item__name"></h3>
+   <h4 class="game-item__popularity"></h4>
+ </div>`;
+
+  const title = item.querySelector('.game-item__name');
+  title.innerText = game.game.name;
```

Comment:
After making changes to `add-games-to-list` you can use `game` here instead of `game.game`.